docs/source/type_inference_and_annotations.rst

```diff
@@ -147,3 +147,14 @@ to be annotated with a starred type:
     p, q, *rs = 1, 2  # type: int, int, *List[int]
 
 Here, the type of ``rs`` is set to ``List[int]``.
+
+Stub Files usage of Ellipsis
```

Comment:
I think this is the right spot to add a section to document what #683 is asking for -- including ellipsis for default arguments. I'd make the heading a little broader: something like "Types in stub files". (Also note the capitalization with sentence-case, which we use in the other headings here.) The thrust of the section would be, remember those stub files we mentioned in another section (and link to the stubs section we have on the "Basics" page); they're normal Python 3, except you generally omit function bodies, variable initializers, and default arguments, using ellipsis instead. With examples for each of those.

https://www.python.org/dev/peps/pep-0484/#stub-files is a helpful reference. I'd be inclined to throw in a mention of the `__getattr__` thing, too, as it seems like a useful feature.